src/freenet/client/filter/CSSTokenizerFilter.java

```diff
@@ -1212,6 +1214,10 @@ else if ("nav-down".equalsIgnoreCase(element)) {
 			elementVerifiers.put(element, new CSSPropertyVerifier(Arrays.asList("contain","cover","fill","none","scale-down"), ElementInfo.VISUALMEDIA));
 			allelementVerifiers.remove(element);
 		}
+		else if("object-position".equalsIgnoreCase(element)) {
+			elementVerifiers.put(element, new CSSPropertyVerifier(null, ElementInfo.VISUALPAGEDMEDIA,null,Arrays.asList("4a5a40","4a40 5a40","5a40 4a40","4 40 5 40","5 40 4 40")));
```

Comment:
as far as I can tell the allowed values for `object-position` should be the same as for `background-position`.

- https://developer.mozilla.org/en-US/docs/Web/CSS/object-position#syntax
- https://developer.mozilla.org/en-US/docs/Web/CSS/background-position#syntax

The implementation here is different, though. Can you double-check and harmonize both? Given the complexity of this value, they may need more tests.

Reply:
background-position filter is broken, so I'll just use the new object-position for both.